neuralpp/test/quick_tests/symoblic/simplication_test.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tests of SymPy's simplication functions.
+https://docs.sympy.org/latest/tutorial/simplification.html
+"""
+from sympy import *
+
+
+def test_simplify() -> None:
+    """
+    One can just use sympy.simplify().
+    It's push-button but has 2 caveats:
+    1. "simplest" form is not well-defined. (showcased by last set of asserts)
+    2. it can be unnecessarily slow due to generality.
+    """
+    x = symbols('x')
+    assert simplify(sin(x)**2 + cos(x)**2) == 1
+    assert simplify((x**3 + x**2 - x - 1)/(x**2 + 2*x + 1)) == x - 1
+    assert simplify(gamma(x) / gamma(x - 2)) == (x - 2) * (x - 1)
+    # we may want to simplify (x**2 + 2*x + 1) into (x+1)**2 but that's not "simplest" recognized by simplify()
```

Comment:
It's very nice that you are using the tests to illustrate the finer points like that!